RecoBTag/FeatureTools/plugins/DeepFlavourTagInfoProducer.cc

```diff
@@ -152,6 +152,15 @@ void DeepFlavourTagInfoProducer::produce(edm::Event& iEvent, const edm::EventSet
 
   edm::Handle<ShallowTagInfoCollection> shallow_tag_infos;
   iEvent.getByToken(shallow_tag_info_token_, shallow_tag_infos);
+  double negative_cut = 0;//used only with flip_
+  if (flip_){//FIXME: Check if can do even less often than once per event
```

Comment:
The framework guarantees that provenance is stable over an entire Run

Reply:
nice to know.
My guess was bordering on some IOV possible boundaries.
It sounds like the beginRun can be the more safe and complete version of this code.
Thanks.